frontend/packages/core/src/AppLayout/notifications.tsx

```diff
@@ -111,7 +111,7 @@ const Notifications: React.FC<NotificationsProp> = ({ data }) => {
                 <MenuList
                   autoFocusItem={open}
                   id="notification-options"
-                  onKeyDown={handleListKeyDown}
+                  onKeyDown={e => handleListKeyDown(e)}
```

Comment:
this is surprising, what lint rule is this?

Reply:
Adjusted these, didn't realize it was because `handleListKeyDown` was a function.